src/likechain/iscn/amino.ts

```diff
@@ -0,0 +1,81 @@
+import { Buffer } from 'buffer/';
+import {
+  IscnRecord,
+  MsgCreateIscnRecord,
+  MsgUpdateIscnRecord,
+  MsgChangeIscnRecordOwnership,
+} from './tx';
+
+function recordToAmino(record?: IscnRecord): any {
+  const result: any = {
+    ...record,
+    stakeholders: record?.stakeholders.map(s => JSON.parse(s.toString())),
+    contentMetadata: JSON.parse(record?.contentMetadata.toString() || '{}'),
+  };
+  if (!result.stakeholders || result.stakeholders.length === 0) {
+    delete result.stakeholders;
```

Comment:
should we keep empty array if `result.stakeholders.length === 0` ?

Reply:
No, since there is `omitempty` in the Go struct definition for JSON serialization, keeping the empty array will result in wrong sign doc.
(same for `contentFingerprints`)